frontend/src/views/Pages/AccountOverview.vue

```diff
@@ -110,7 +110,7 @@ export default {
       this.loading = false
     },
     onReset() {
-      this.transactionData = EMPTY_TRANSACTION_DATA
+      this.transactionData = { ...EMPTY_TRANSACTION_DATA }
```

Comment:
I am happy to have found this by testing!

Reply:
Explain, enlighten me aswell pls ❤️